include/rocksdb/env.h

```diff
@@ -530,12 +533,19 @@ class Env {
 
   virtual void SanitizeEnvOptions(EnvOptions* /*env_opts*/) const {}
 
+  // Get the FileSystem implementation this Env was constructed with. It
+  // could be a fully implemented one, or a wrapper class around the Env
+  std::shared_ptr<FileSystem> GetFileSystem();
```

Comment:
shouldn't this be const std::shared_ptr<FileSystem>& GetFileSystem() const?

Reply:
Makes sense, since we can avoid some overhead if the caller just wants to dereference it immediately.